tests/common/common.h

```diff
@@ -148,25 +148,23 @@ void check_enum_underlying_type(sycl_cts::util::logger& log) {
 /**
  * @brief Helper function to check an info parameter.
  */
-template <typename enumT, typename returnT, enumT kValue, typename objectT>
+template <typename infoDesc, typename returnT, typename objectT>
```

Comment:
Usually the template parameters are capitalized
```suggestion
template <typename InfoDesc, typename ReturnT, typename ObjectT>
```